master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -178,33 +178,34 @@ PING google.com (216.58.219.206): 56 data bytes
 ### Prevent outgoing connections from pods
 
 Kubernetes NetworkPolicy does not provide a way to prevent outgoing connections from pods.  However,
-Calico does. In this section we'll create a Policy using `calicoctl` which prevents all outgoing
-connections from Kubernetes pods in the advanced-policy-demo Namespace.
+Calico does. In this section we'll create egress Policies using `calicoctl`
+which allow the outgoing connections in the advanced-policy-demo Namespace we
```

Comment:
"which" --> "that"

backticks around "advanced-policy-demo" : `advanced-policy-demo`

"Namespace" --> "namespace" (not a proper noun)